avocado/utils/crypto.py

```diff
@@ -66,11 +66,11 @@ def hash_file(filename, size=None, algorithm="md5"):
         while size > 0:
```

Comment:
The try/except clause above is broken. Mind fixing (in a separate commit)? Just add a `return None` right after the `logging.error(Unknown...`

Reply:
Ah, sure, will fix it.